charts/egeria-platform/values.yaml

```diff
@@ -0,0 +1,117 @@
+# Egeria Settings
+egeria:
+  # Global Egeria Settings
+  logging: error
+  version: "3.5"
+  debug: true
+
+  # persistence
+  persistence: true
+  storageClass: ibmc-vpc-block-10iops-tier
+  storageSize: 8Gi
+  repositoryType: "local-graph-repository"
+
+  # communication
+  cohort: mds
+  viewOrg: org
+  kafkaEndpoint: egeria-kafka:9092
+
+  # userid used for admin ops (no security enabled by default)
+  user: garygeeke
+
+  # list of servers that should start on this platform
+  # the names must correspond to a config-<name>.sh script
+  # as well as to the actual instance name
+  serverList: mds1,view1
+
+  # OMAG Server
+  # usually you need to configure a connection to an OMAG server
+  # the values below will result in EGERIA_OMAG_SERVER_NAME and EGERIA_OMAG_SERVER_URL environment variables
+  # if platformUrl is empty this will point to itself
+  omagServer:
+    name: mds1
+    platformUrl: ""
+    
+
+# Additional environment variables that are needed for setup the servers
+extraEnv: {}
+
+# name a secret to mount as environment variables to store secrets
+secretName: ""
+
+replicaCount: 1
+
+image:
+  repository: quay.io/odpi/egeria
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "3.5-SNAPSHOT"
+  
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+  
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 0
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  platformPort: 9443
+  debugPort: 5005
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
```

Comment:
why is this not commented out, but other entries are?
ingress is quite hard to get right generically.. so ideas welcome!

Reply:
This is the default output for `helm init`